packages/shared/lib/core/profile/actions/active-profile/login.ts

```diff
@@ -17,6 +18,9 @@ import { recoverAndLoadAccounts } from './recoverAndLoadAccounts'
 export async function login(firstTime: boolean = false): Promise<void> {
     const { loggedIn, lastActiveAt, id, isStrongholdLocked, type } = get(activeProfile)
     if (id) {
+        loggedIn.set(true)
+        await getAndUpdateNodeInfo()
+
```

Comment:
Added this so the HRP is loaded before we load the activities. We need to rethink the networking a bit so we use the data from the node more and persist this between sessions on the profile object.